### DISCUSSION TITLE
Is it possible to use IMAP instead of SMTP?

### DISCUSSION THREAD

**Discussion by felixriehm** [mastodon/mastodon#19667] [Q&A] [answered]
When following the [installtion guide](https://docs.joinmastodon.org/admin/install/) you will at some point run the interactive setup wizard. There you can specify a SMTP server. Is it possible to use IMAP instead?

**Comment by alex9099** [comment, marked as answer]
SMTP is a protocol used to send emails, IMAP is only used to gather emails from the email provider to your mailbox .

Mastodon requires SMTP to send mails to users.

I'm what case would you want to use IMAP instead of SMTP?

**Reply by felixriehm** [reply]
Thanks for the info.
